package.json

```diff
@@ -162,7 +162,34 @@
       "testRule": true
     },
     "rules": {
-      "no-mixed-spaces-and-tabs": 0
+      "array-callback-return": "error",
+      "dot-notation": "error",
+      "func-name-matching": "error",
+      "guard-for-in": "error",
+      "no-else-return": [
+        "error",
+        {
+          "allowElseIf": false
+        }
+      ],
+      "no-implicit-coercion": "error",
+      "no-lonely-if": "error",
+      "no-mixed-spaces-and-tabs": "off",
+      "no-unneeded-ternary": "error",
+      "prefer-arrow-callback": "off",
+      "prefer-const": "error",
+      "prefer-destructuring": [
+        "off",
+        {
+          "object": true,
+          "array": false
+        }
+      ],
+      "prefer-object-spread": "error",
+      "prefer-regex-literals": "error",
+      "prefer-rest-params": "error",
+      "prefer-spread": "error",
+      "prefer-template": "off"
```

Comment:
```suggestion
```

This will conflict with #4429.

Reply:
I know it will conflict, it's whatever gets merged. I will rebase it anyway